<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NFMDemodGUI</class>
 <widget class="RollupWidget" name="NFMDemodGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>364</width>
    <height>200</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>364</width>
    <height>200</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Liberation Sans</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>NFM Demodulator</string>
  </property>
  <widget class="QWidget" name="settingsContainer" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>362</width>
     <height>191</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>362</width>
     <height>0</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Settings</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>3</number>
    </property>
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="deltaFreqPowerLayout">
      <property name="topMargin">
       <number>2</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="DeltaFrequencyLayout">
        <item>
         <widget class="QLabel" name="deltaFrequencyLabel">
          <property name="minimumSize">
           <size>
            <width>16</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Df</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="ValueDialZ" name="deltaFrequency" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>32</width>
            <height>16</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Liberation Mono</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="focusPolicy">
           <enum>Qt::StrongFocus</enum>
          </property>
          <property name="toolTip">
           <string>Demod shift frequency from center in Hz</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="deltaUnits">
          <property name="text">
           <string>Hz </string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="channelPower">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Channel power</string>
        </property>
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="text">
         <string>-100.0 dB</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="levelMeterLayout">
      <item>
       <widget class="QLabel" name="channelPowerMeterUnits">
        <property name="text">
         <string>dB</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="LevelMeterSignalDB" name="channelPowerMeter" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Liberation Mono</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string>Level meter (dB) top trace: average, bottom trace: instantaneous peak, tip: peak hold</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="rfBWLayout">
      <item>
       <widget class="QPushButton" name="channelSpacingApply">
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Apply channel spacing</string>
        </property>
        <property name="text">
         <string>CS</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="channelSpacing">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Channel spacing selection</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="channelSpacingUnits">
        <property name="text">
         <string>k  </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="rfLabel">
        <property name="text">
         <string>RF</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="rfBW">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>RF bandwidth (kHz)</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>400</number>
        </property>
        <property name="pageStep">
         <number>1</number>
        </property>
        <property name="value">
         <number>110</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="rfBWText">
        <property name="minimumSize">
         <size>
          <width>35</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>20.0k</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="bwSpacer">
        <property name="minimumSize">
         <size>
          <width>10</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="afLabel">
        <property name="text">
         <string>AF</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="afBW">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Audio bandwidth (kHz)</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>200</number>
        </property>
        <property name="pageStep">
         <number>1</number>
        </property>
        <property name="value">
         <number>30</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="afBWText">
        <property name="minimumSize">
         <size>
          <width>35</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>20.0k</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="squelchLayout">
      <item>
       <widget class="QLabel" name="fmDevLabel">
        <property name="text">
         <string>FMd</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="fmDev">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>FM peak deviation (kHz)</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>200</number>
        </property>
        <property name="pageStep">
         <number>1</number>
        </property>
        <property name="value">
         <number>25</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="fmDevText">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>+20.0k</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="squelchLabel">
        <property name="text">
         <string>  Sq</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ButtonSwitch" name="deltaSquelch">
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="acceptDrops">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string>Toggle AF balance (on) or channel power (off) based squelch</string>
        </property>
        <property name="text">
         <string>D</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDial" name="squelch">
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Squelch threshold (dB)</string>
        </property>
        <property name="minimum">
         <number>-100</number>
        </property>
        <property name="maximum">
         <number>0</number>
        </property>
        <property name="singleStep">
         <number>1</number>
        </property>
        <property name="pageStep">
         <number>1</number>
        </property>
        <property name="value">
         <number>-100</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="squelchText">
        <property name="minimumSize">
         <size>
          <width>28</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Squelch threshold (dB)</string>
        </property>
        <property name="text">
         <string>-100</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDial" name="squelchGate">
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Squelch gate (ms)</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>50</number>
        </property>
        <property name="pageStep">
         <number>1</number>
        </property>
        <property name="value">
         <number>5</number>
        </property>
        <property name="sliderPosition">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="squelchGateText">
        <property name="minimumSize">
         <size>
          <width>25</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Squelch gate (ms)</string>
        </property>
        <property name="text">
         <string>000</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="ctcssLayout">
      <item>
       <widget class="QCheckBox" name="ctcssOn">
        <property name="toolTip">
         <string>Activate CTCSS</string>
        </property>
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="text">
         <string>CTCSS</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="ctcss">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Set CTCSS Frequency</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="ctcssText">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>CTCSS detected</string>
        </property>
        <property name="text">
         <string>--</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="dcsOn">
        <property name="toolTip">
         <string>Activate DCS</string>
        </property>
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="text">
         <string>DCS</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="dcsCode">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Set DCS code</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="dcsPositive">
        <property name="acceptDrops">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string>Display DCS code as postive/normal (+) or negative/inverted (-)</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../sdrgui/resources/res.qrc">
          <normaloff>:/minusw.png</normaloff>
          <normalon>:/plusw.png</normalon>:/minusw.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="dcsText">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>DCS detected</string>
        </property>
        <property name="text">
         <string>--</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="audioLayout">
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="volumeLabel">
        <property name="text">
         <string>Vol</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDial" name="volume">
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Sound volume (%)</string>
        </property>
        <property name="maximum">
         <number>200</number>
        </property>
        <property name="singleStep">
         <number>1</number>
        </property>
        <property name="pageStep">
         <number>1</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="volumeText">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Sound volume (%)</string>
        </property>
        <property name="text">
         <string>100</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ButtonSwitch" name="highPassFilter">
        <property name="toolTip">
         <string>High pass audio filter</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../sdrgui/resources/res.qrc">
          <normaloff>:/filter_highpass.png</normaloff>:/filter_highpass.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="audioMute">
        <property name="toolTip">
         <string>Left: Mute/Unmute audio Right: Select audio output device</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../../sdrgui/resources/res.qrc">
          <normaloff>:/sound_on.png</normaloff>
          <normalon>:/sound_off.png</normalon>:/sound_on.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RollupWidget</class>
   <extends>QWidget</extends>
   <header>gui/rollupwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ButtonSwitch</class>
   <extends>QToolButton</extends>
   <header>gui/buttonswitch.h</header>
  </customwidget>
  <customwidget>
   <class>ValueDialZ</class>
   <extends>QWidget</extends>
   <header>gui/valuedialz.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LevelMeterSignalDB</class>
   <extends>QWidget</extends>
   <header>gui/levelmeter.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../sdrgui/resources/res.qrc"/>
 </resources>
 <connections/>
</ui>
